specification/dartLangSpec.tex

```diff
@@ -14415,15 +14415,18 @@ \subsection{Imports}
 Any import that is not deferred is immediate.
 
 \LMHash{}%
-It is a compile-time error if the specified URI of an immediate import does not refer to a library declaration.
+It is a compile-time error if the specified URI of an import does not
+refer to a library declaration.
 The interpretation of URIs is described in section \ref{uris} below.
 
 \LMHash{}%
-It is a compile-time error if the specified URI of a deferred import does not refer to a library declaration.
+It is a dynamic error if the specified URI of a deferred import does not
+refer to a library declaration when the library is loaded.
 
-\rationale{
-One cannot detect the problem at compile time because compilation often occurs during execution and one does not know what the URI refers to.
-However the development environment should detect the problem.
+\rationale{%
+The problem cannot be detected at compile time because
+one does not know what the URI refers to when the library is loaded.
```

Comment:
I'd instead say that:

> It's a run-time error if the `loadLibrary` call on a deferred import cannot load the library that was specified by the import URI at compile-time.

Then we make it entirely implementation dependent how to do things, but we do require that there is only one library.

Reply:
That's a nice way to avoid talking about the "signature" of the library etc. Done!